### PR TITLE
Pull Request Decorator debug

### PR DESCRIPTION
**Pull request by TsuyoshiUshio**

### CODE REVIEW

File: VulnerableApplication/Global.asax.cs [hunk 1]
```diff
@@ -17,5 +17,7 @@ protected void Application_Start()
             RouteConfig.RegisterRoutes(RouteTable.Routes);
             BundleConfig.RegisterBundles(BundleTable.Bundles);
         }
+        
+        public const string Password = "abc";
```
Comment (TsuyoshiUshio) [inline VulnerableApplication/Global.asax.cs:21]: **CODE_SMELL**
> 3 more lines of code need to be covered by tests to reach the minimum threshold of 65.0% lines coverage.
 See [details](https://sonarcloud.io/project/issues?id=TsuyoshiUshio_VulnerableApp&open=AWure2BHojvkCqRLlWU5&pullRequest=16&resolved=false)

Comment (TsuyoshiUshio) [inline VulnerableApplication/Global.asax.cs:21]: **CODE_SMELL**
> 7 more comment lines need to be written to reach the minimum threshold of 25.0% comment density.
 See [details](https://sonarcloud.io/project/issues?id=TsuyoshiUshio_VulnerableApp&open=AWure2BHojvkCqRLlWU6&pullRequest=16&resolved=false)

Comment (TsuyoshiUshio) [inline VulnerableApplication/Global.asax.cs:21]: **CODE_SMELL**
> Add or update the header of this file.
 See [details](https://sonarcloud.io/project/issues?id=TsuyoshiUshio_VulnerableApp&open=AWure2BHojvkCqRLlWU7&pullRequest=16&resolved=false)

Comment (TsuyoshiUshio) [inline VulnerableApplication/Global.asax.cs:21]: **VULNERABILITY**
> 'password' detected in this expression, review this potentially hardcoded credential.
 See [details](https://sonarcloud.io/project/issues?id=TsuyoshiUshio_VulnerableApp&open=AWure2BHojvkCqRLlWU8&pullRequest=16&resolved=false)

Comment (TsuyoshiUshio) [inline VulnerableApplication/Global.asax.cs:21]: **CODE_SMELL**
> Change this constant to a 'static' read-only property.
 See [details](https://sonarcloud.io/project/issues?id=TsuyoshiUshio_VulnerableApp&open=AWure2BHojvkCqRLlWU9&pullRequest=16&resolved=false)

**Comment by TsuyoshiUshio** [comment]
**CODE_SMELL**
> [VulnerableApplication] Mark this assembly with 'System.CLSCompliantAttribute'
 See [details](https://sonarcloud.io/project/issues?id=TsuyoshiUshio_VulnerableApp&open=AWure2B6ojvkCqRLlWU-&pullRequest=16&resolved=false)

File: VulnerableApplication/Global.asax.cs (hunk 1 above)
Comment (TsuyoshiUshio) [reply VulnerableApplication/Global.asax.cs:21]: /workItem

Comment (TsuyoshiUshio) [reply VulnerableApplication/Global.asax.cs:21]: /workItem

Comment (TsuyoshiUshio) [reply VulnerableApplication/Global.asax.cs:21]: /workItem

Comment (TsuyoshiUshio) [reply VulnerableApplication/Global.asax.cs:21]: /workItem

Comment (TsuyoshiUshio) [reply VulnerableApplication/Global.asax.cs:21]: /workItem

Comment (TsuyoshiUshio) [reply VulnerableApplication/Global.asax.cs:21]: WorkItem 808 Created see [workItem](https://dev.azure.com/csedevops/ea38b5b8-2b4f-4721-8514-fe765aa4a23a/_workitems/edit/808).

Comment (TsuyoshiUshio) [reply VulnerableApplication/Global.asax.cs:21]: /falsepositive

Comment (TsuyoshiUshio) [reply VulnerableApplication/Global.asax.cs:21]: /falsepositive

Comment (TsuyoshiUshio) [reply VulnerableApplication/Global.asax.cs:21]: The issue marked as falsepositive. For more [detail](https://sonarcloud.io/project/issues?id=TsuyoshiUshio_VulnerableApp&open=AWure2BHojvkCqRLlWU5&pullRequest=16&resolved=false).

Comment (TsuyoshiUshio) [reply VulnerableApplication/Global.asax.cs:21]: /falsepositive

Comment (TsuyoshiUshio) [reply VulnerableApplication/Global.asax.cs:21]: /falsepositive

Comment (TsuyoshiUshio) [reply VulnerableApplication/Global.asax.cs:21]: /falsepositive

Comment (TsuyoshiUshio) [reply VulnerableApplication/Global.asax.cs:21]: /falsepositive

Comment (TsuyoshiUshio) [reply VulnerableApplication/Global.asax.cs:21]: The issue marked as falsepositive. For more [detail](https://sonarcloud.io/project/issues?id=TsuyoshiUshio_VulnerableApp&open=AWure2BHojvkCqRLlWU5&pullRequest=16&resolved=false).

Comment (TsuyoshiUshio) [reply VulnerableApplication/Global.asax.cs:21]: /falsepositive

Comment (TsuyoshiUshio) [reply VulnerableApplication/Global.asax.cs:21]: The issue marked as falsepositive. For more [detail](https://sonarcloud.io/project/issues?id=TsuyoshiUshio_VulnerableApp&open=AWure2BHojvkCqRLlWU5&pullRequest=16&resolved=false).

Comment (TsuyoshiUshio) [reply VulnerableApplication/Global.asax.cs:21]: The issue marked as falsepositive. For more [detail](https://sonarcloud.io/project/issues?id=TsuyoshiUshio_VulnerableApp&open=AWure2BHojvkCqRLlWU6&pullRequest=16&resolved=false).

Comment (TsuyoshiUshio) [reply VulnerableApplication/Global.asax.cs:21]: /falsepositive

Comment (TsuyoshiUshio) [reply VulnerableApplication/Global.asax.cs:21]: The issue marked as falsepositive. For more [detail](https://sonarcloud.io/project/issues?id=TsuyoshiUshio_VulnerableApp&open=AWure2BHojvkCqRLlWU7&pullRequest=16).

Comment (TsuyoshiUshio) [reply VulnerableApplication/Global.asax.cs:21]: Just hello.

Comment (TsuyoshiUshio) [reply VulnerableApplication/Global.asax.cs:21]: /daruyanagi

Comment (TsuyoshiUshio) [reply VulnerableApplication/Global.asax.cs:21]: /falsepositive

Comment (TsuyoshiUshio) [reply VulnerableApplication/Global.asax.cs:21]: /falsepositive

Comment (TsuyoshiUshio) [reply VulnerableApplication/Global.asax.cs:21]: /falsepositive

Comment (TsuyoshiUshio) [reply VulnerableApplication/Global.asax.cs:21]: The issue is already resolved or confirmed.

Comment (TsuyoshiUshio) [reply VulnerableApplication/Global.asax.cs:21]: /falsepositive

Comment (TsuyoshiUshio) [reply VulnerableApplication/Global.asax.cs:21]: /falsepositve

Comment (TsuyoshiUshio) [reply VulnerableApplication/Global.asax.cs:21]: /falsepositive

Comment (TsuyoshiUshio) [reply VulnerableApplication/Global.asax.cs:21]: The issue is already resolved or confirmed.

Comment (TsuyoshiUshio) [reply VulnerableApplication/Global.asax.cs:21]: /falsepositive

Comment (TsuyoshiUshio) [reply VulnerableApplication/Global.asax.cs:21]: /falsepositive

Comment (TsuyoshiUshio) [reply VulnerableApplication/Global.asax.cs:21]: The issue marked as falsepositive. For more [detail](https://sonarcloud.io/project/issues?id=TsuyoshiUshio_VulnerableApp&open=AWure2BHojvkCqRLlWU9&pullRequest=16).

Comment (TsuyoshiUshio) [reply VulnerableApplication/Global.asax.cs:21]: The issue is already resolved or confirmed.
